feature-group-definitions/class-syntax.yml

```diff
@@ -0,0 +1,9 @@
+caniuse: es6-class
```

Comment:
There are many ways to name this feature, but yeah I think `class-syntax` works well.